aiida/backends/tests/cmdline/commands/test_import.py

```diff
@@ -85,8 +88,67 @@ def test_comment_mode(self):
         result = self.cli_runner.invoke(cmd_import.cmd_import, options)
         self.assertIsNone(result.exception, result.output)
         self.assertIn('Comment mode: newest', result.output)
+        self.assertEqual(result.exit_code, 0, result.output)
 
         options = ['--comment-mode', 'overwrite'] + archives
         result = self.cli_runner.invoke(cmd_import.cmd_import, options)
         self.assertIsNone(result.exception, result.output)
         self.assertIn('Comment mode: overwrite', result.output)
+        self.assertEqual(result.exit_code, 0, result.output)
+
+    def test_import_old_local_archives(self):
+        """ Test import of old local archives
+        Expected behavior: Return message in terminal and error code != 0
+        """
+        archives = [('export/migrate/export_v0.1.aiida', '0.1'), ('export/migrate/export_v0.2.aiida', '0.2'),
+                    ('export/migrate/export_v0.3.aiida', '0.3')]
+
+        for archive, version in archives:
+            options = [get_archive_file(archive)]
+            result = self.cli_runner.invoke(cmd_import.cmd_import, options)
+
+            self.assertIsNotNone(result.exception, result.output)
+            self.assertNotEqual(result.exit_code, 0, result.output)
+            self.assertIn(version, result.output, result.exception)
+
+    def test_import_old_url_archives(self):
+        """ Test import of old URL archives
+        Expected behavior: Return message in terminal and error code != 0
+        """
+        url = "https://raw.githubusercontent.com/aiidateam/aiida_core/f685844b290ed23df254873df62da7162bde0f1b/"
+        archive_path = "aiida/backends/tests/fixtures/export/migrate/"
+        archives = [('export_v0.1.aiida', '0.1'), ('export_v0.2.aiida', '0.2'), ('export_v0.3.aiida', '0.3')]
```

Comment:
Fine to test the import via URL once; no need to do requests for all versions (it's already tested above)

Reply:
Sure, will reduce.